.gitignore

```diff
@@ -25,3 +25,4 @@ contrib/examples/consumer/Gopkg.lock
 contrib/examples/consumer
 contrib/examples/vendor/*
 integration.test*
+debug.test*
```

Comment:
where is this coming from?

Reply:
Was some kind of binary that showed up after running a failed make command, I can't remember which. I'm assuming we would prefer not to have that checked in.

Reply:
Yeah I've seen those pop up too after running the tests.